core/layout_api.php

```diff
@@ -397,14 +397,14 @@ function layout_navbar() {
 
 	# mobile view
 	echo '<div class="hidden-sm hidden-md hidden-lg">';
-	echo '<nav class="navbar-menu pull-left navbar-collapse collapse" role="navigation" style="height: auto;">';
```

Comment:
@dregad 
No. I would switch "<nav" with "<div" instead. I just tried it and it seems to work just fine.